axelor-account/src/main/resources/domains/AccountConfig.xml

```diff
@@ -179,8 +179,8 @@
       default="true"/>
     <boolean name="fixedAssetCatReqOnInvoice"
       title="Fixed asset category required on invoices"/>
-    <boolean name="isPrintInvoicesInCompanyLanguage"
-      title="Print invoices in company language" default="true"/>
+    <boolean name="isPrintInvoicesInCompanyLocalization"
+      title="Print invoices in company localization" default="true"/>
```

Comment:
Just change the title, not the fields name, this is supposed to be an anomaly, we don't do schema changes in anomalies.